deployments/mainnet/types/factories/contracts/bancor-portal/BancorPortalV1__factory.ts

```diff
@@ -0,0 +1,651 @@
+/* Autogenerated file. Do not edit manually. */
+
+/* tslint:disable */
+
+/* eslint-disable */
+import type {
+  BancorPortal,
+  BancorPortalInterface,
+} from "../../../contracts/bancor-portal/BancorPortal";
```

Comment:
You did everything right, other than this tricky (and currently WIP/not documented) change: it should be `"../../../contracts/bancor-portal/BancorPortalV1"` so that the factory references the deprecated type 😅 😅 😅

Reply:
done